package.json

```diff
@@ -16,9 +16,11 @@
   "dependencies": {
     "argv-split": "^1.0.0",
     "gemini-scrollbar": "^1.4.3",
+    "iconv-lite": "^0.4.13",
     "octicons": "^4.3.0",
     "pikaday": "^1.4.0",
-    "sigma": "^1.0.3"
+    "sigma": "^1.0.3",
+    "tmp": "0.0.28"
```

Comment:
Definitely – I just didn't know `devDependencies` existed, this is my first time touching Node. Done!